src/app/search/search-filters.ts

```diff
@@ -533,130 +490,119 @@ function searchFilters(
       // http://blog.tatedavies.com/2012/08/28/replace-microsoft-chars-in-javascript/
       query = query.replace(/[\u2018|\u2019|\u201A]/g, "'");
       query = query.replace(/[\u201C|\u201D|\u201E]/g, '"');
-      // could probably tidy this regex, just a quick hack to support multi term:
-      // [^\s]*?"[^"]+?" -> match is:"stuff here"
-      // [^\s]*?'[^']+?' -> match is:'stuff here'
-      // [^\s"']+' -> match is:stuff
-      const searchTerms = query.match(/[^\s]*?"[^"]+?"|[^\s]*?'[^']+?'|[^\s"']+/g) || [];
+      // \S*?(["']).*?\1 -> match `is:"stuff here"` or `is:'stuff here'`
+      // [^\s"']+ -> match is:stuff
+      const searchTerms = query.match(/\S*?(["']).*?\1|[^\s"']+/g) || [];
```

Comment:
This is one place where I wish we had unit tests - this regex has been really tricky to get right. I'll trust that you've tested this appropriately, but some issues we've had before includes quotes inside quotes.

Reply:
this accounts for being able to search `"` by using '`'s, or vice versa, and parsed my test cases of multiple quotes etc. it is nothing fancy just the existing regex with the same greed/laziness, but with a backreference